src/test/java/de/unistuttgart/overworldbackend/MinigameInputControllerTest.java

```diff
@@ -249,4 +249,94 @@ void submitGameData_MinigameDoesNotExist_ThrowNotFound() throws Exception {
       .perform(post(fullURL + "/submit-game-pass").content(bodyValue).contentType(MediaType.APPLICATION_JSON))
       .andExpect(status().isNotFound());
   }
+
+  @Test
+  void createLectureAndSubmitMinigameData() throws Exception {
+    String currentUrl = "/courses";
+    final CourseInitialData toCreateCourse = new CourseInitialData("testName", "SS-22", "testDescription");
+    final String bodyValueCourse = objectMapper.writeValueAsString(toCreateCourse);
+
+    final MvcResult resultCourse = mvc
+      .perform(post(currentUrl).content(bodyValueCourse).contentType(MediaType.APPLICATION_JSON))
+      .andExpect(status().isCreated())
+      .andReturn();
+
+    final CourseDTO createdCourse = objectMapper.readValue(
+      resultCourse.getResponse().getContentAsString(),
+      CourseDTO.class
+    );
+
+    MinigameTaskDTO updateMinigameTaskDTO = createdCourse
+      .getWorlds()
+      .get(0)
```

Comment:
Perhaps you should extract `createdCourse.getWorlds().get(0)` into a variable... you use it more than once.